src/components/util/resolve-cells.js

```diff
@@ -0,0 +1,35 @@
+import React, { Children } from "react";
+import omit from "lodash.omit";
+import merge from "lodash.merge";
+import resolveCellDefaults from "./resolve-cell-defaults";
+import resolveColumnCounts from "./resolve-column-counts";
+import resolveCellStyles from "./resolve-cell-styles";
+
+const resolveCells = (props) => {
+  // Resolve the final style defaults for each cell
+  const childProps = omit(props, ["children", "style"]);
```

Comment:
Unless we actually need deep merging, which we very well might, huh? Either way, we should be careful of `merge` when the faster `assign` will do.